python/runtime/dbapi/pyalisa/client.py

```diff
@@ -168,12 +166,12 @@ def read_logs(self, task_id, offset, w):
             params["AlisaTaskId"] = task_id
             params["Offset"] = str(offset)
             log = self._requet_and_parse_response("GetAlisaTaskLog", params)
```

Comment:
_requet_and_parse_response -> _request_and_parse_response
Sorry, this is a legacy problem. Please refine it on next commit.